db/models.py

```diff
@@ -1 +1,32 @@
 from django.db import models
+
+
+class Race(models.Model):
+    RACE_CHOICE = [
+        ("Elf", "Elf"),
+        ("Dwarf", "Dwarf"),
+        ("Human", "Human"),
+        ("Ork", "Ork")
+    ]
+    name = models.CharField(max_length=255, unique=True, choices=RACE_CHOICE)
```

Comment:
Consider adding a 'verbose_name' to the 'name' field for better readability in the admin interface.